<?xml version="1.0"?>
<launch>
    <arg name="hand_id" default="rh"/>
    <arg name="fingers" default="ff"/>
    <node name="sr_$(arg hand_id)_piezo_feedback" pkg="sr_piezo_feedback" type="sr_piezo_feedback_node.py" output="screen">
        <param name="/hand_id" value="$(arg hand_id)"/>
        <param name="/fingers" value="$(arg fingers)"/>
    </node>
</launch>